rosettastmtont/.htaccess

```diff
@@ -1,15 +1,31 @@
-Options +FollowSymLinks
-RewriteEngine on
+RewriteEngine On
+RewriteBase /rosettastmtont/
 
-# Redirect the base URL to the latest version
-RewriteRule ^$ https://raw.githubusercontent.com/karanthpallavi/rosetta_statement_ontology/main/rosettastmtont-full.owl [R=303,L]
+# ---- Ontology Landing Page ----
+RewriteRule ^$ https://karanthpallavi.github.io/rosetta_statement_ontology/ [R=302,L]
 
-# Redirect terms (e.g., classes, properties)
-#RewriteRule ^(.+)$ https://raw.githubusercontent.com/karanthpallavi/rosetta_statement_ontology/main/$1 [R=303,L]
+# ---- Ontology File (latest/latest format) ----
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^ontology$ https://karanthpallavi.github.io/rosetta_statement_ontology/rosettastmtont.owl [R=303,L]
 
-# Redirect all ontology terms (classes, properties, individuals) to term fragment 
-RewriteRule ^([^/]+)$ https://raw.githubusercontent.com/karanthpallavi/rosetta_statement_ontology/main/rosettastmtont-full.owl#$1 [R=303,L]
+RewriteCond %{HTTP_ACCEPT} application/ld\+json
+RewriteRule ^ontology$ https://karanthpallavi.github.io/rosetta_statement_ontology/rosettastmtont.json [R=303,L]
 
-# Redirect specific versions when they are available
-#RewriteRule ^v1.0$ https://raw.githubusercontent.com/karanthpallavi/rosetta_statement_ontology/v1.0/rosettastmtont-full.owl [R=303,L]
-#RewriteRule ^v2.0$ https://raw.githubusercontent.com/karanthpallavi/rosetta_statement_ontology/v2.0/rosettastmtont-full.owl [R=303,L]
+# Fallback to HTML landing page
+RewriteRule ^ontology$ https://karanthpallavi.github.io/rosetta_statement_ontology/ [R=303,L]
+
+# ---- Versioned Ontology Files ----
+RewriteRule ^ontology/releases/(.*)/rosettastmtont.owl$ https://karanthpallavi.github.io/rosetta_statement_ontology/releases/$1/rosettastmtont.owl [R=302,L]
+RewriteRule ^ontology/releases/(.*)/rosettastmtont.json$ https://karanthpallavi.github.io/rosetta_statement_ontology/releases/$1/rosettastmtont.json [R=302,L]
+
+# ---- Individual Terms (Classes, Properties, Individuals) ----
+# RDF/XML
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^(.+)$ https://karanthpallavi.github.io/rosetta_statement_ontology/terms/$1.owl [R=303,L]
```

Comment:
i think this will not work as you intend, since you don't have a file per class in the target repository. Are you sure you don't want to redirect to an owl file directly?